geopandas/io/tests/test_file.py

```diff
@@ -340,11 +340,35 @@ def test_read_file_filtered__rows_bbox__polygon(df_nybb):
     assert filtered_df_shape == (1, 5)
 
 
-def read_file_filtered_rows_invalid():
+def test_read_file_filtered_rows_invalid():
     with pytest.raises(TypeError):
         read_file(geopandas.datasets.get_path("nybb"), rows="not_a_slice")
 
 
+@pytest.mark.skipif(
+    LooseVersion(fiona.__version__) < LooseVersion("1.8"),
+    reason="Ignore geometry only available in Fiona 1.8",
+)
+def test_read_file__ignore_geometry():
+    pdf = geopandas.read_file(
+        geopandas.datasets.get_path("naturalearth_lowres"), ignore_geometry=True,
+    )
+    assert "geometry" not in pdf.columns
+    assert isinstance(pdf, pd.DataFrame)
```

Comment:
GeoDataFrame returns True here as it is based on DataFrame. I assume you want to check if it returned pandas DataFrame instead, right? We may do `isinstance(pdf, pd.DataFrame) and not isinstance(pdf, gpd.GeoDataFrame)`.

Reply:
That is a good catch there, I will fix it in a bit.